mail_template_conditional_attachment/models/mail_template.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Druidoo - Iván Todorovich <ivan.todorovich@druidoo.io>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models, fields
+
+
+class MailTemplate(models.Model):
+    _inherit = 'mail.template'
+
+    conditional_attachment_ids = fields.One2many(
+        'mail.template.conditional.attachment',
+        'mail_template_id',
+        string='Conditional Attachments',
+    )
+
+    @api.multi
+    def generate_email(self, res_ids, **kwargs):
```

Comment:
I'd say this `**kwargs` should change to be the same as the [parent function](https://github.com/OCA/OCB/blob/10.0/addons/mail/models/mail_template.py#L463), as well as below in line 27 where it's called.